src/lib/DataTable/DataTableRouted.tsx

```diff
@@ -130,16 +147,7 @@ export function DataTableRouted<T, TFilter, TRouteNames>({
       <Table striped hover size="sm" className={tableClassName} style={tableStyle}>
         <thead>
           <tr>
-            {actions &&
-              (actions.columnTitle != null ? (
-                <th className={actions.className} style={actions.style}>
-                  {actions.columnTitle}
-                </th>
-              ) : (
-                <th className={actions.className} style={{ width: "80px", ...actions.style }}>
-                  {dataTableTranslations.actionTitle}
-                </th>
-              ))}
+            {GetActions(ActionsPosition.Left)}
```

Comment:
As mentioned above, call a normal component here. You can than inline your if here:
```
{ actionPosition == ActionsPosition.Left && <YourComponent /> }
```

Reply:
@neotrow Done :)